app/views/experiments/canvas.html.erb

```diff
@@ -1,4 +1,4 @@
-<% provide(:head_title, raw(t("experiments.canvas.head_title", project: @project.name))) %>
+<% provide(:head_title, sanitize_input(t("experiments.canvas.head_title", project: @project.name))) %>
```

Comment:
If we're being pedantic, than @project.name should be escape_input(@project.name), e.g. `sanitize_input(t("experiments.canvas.head_title", project: escape_input(@project.name)))`